runtime/src/bank.rs

```diff
@@ -3890,9 +3892,11 @@ impl Bank {
         }
 
         let mut measure = Measure::start("collect_rent_eagerly-ms");
-        for partition in self.rent_collection_partitions() {
-            self.collect_rent_in_partition(partition);
-        }
+        self.rent_collection_partitions()
+            .into_par_iter()
```

Comment:
I think this would use the full rayon threads, right? Any idea if it might impact other things going on? I suppose there is no transaction processing happening at this point.

Reply:
I think the # of partitions will be small. I think we get > 1 when we have skipped slots? There are things here I don't really understand. I could limit the threads with a thread pool or chunk it or something else.

Reply:
I could add a metric for # partitions, but it should be 1 partition per slot, which should be 1 partition per 400ms, or 2.5 per second, right? If we get behind, it could spike. I think?

Reply:
using acct db's thread pool now.